### DISCUSSION TITLE
RP2040 not visible to other nodes

### DISCUSSION THREAD

**Discussion by pagex** [meshtastic/firmware#5459] [Troubleshooting]
Just installed latest alpha 2.5.14 (tried 2.5.11 as well) on to RP2040 (full erase) while node boots and sends messages (tested via cli) / receives messages as can see in logs packet received, it is not appearing in other nodes list despite being set up as a client tried router as well with same result.

**Comment by pagex** [comment]
Closing as took more then 12 hours for this RP node to appear in other nodes list, which is unusual.

**Comment by fifieldt** [comment]
Thanks a ton for testing the alphas. Not sure why that happened. Does it appear to just be the RP2040? has anyone seen another node type doing this?
